cmd/docker-app/root.go

```diff
@@ -21,8 +23,10 @@ func newRootCmd(dockerCli command.Cli) *cobra.Command {
 			}
 			return nil
 		},
+		Version: fmt.Sprintf("%s, build %s", internal.Version, internal.GitCommit),
 	}
 	cmd.PersistentFlags().BoolVar(&internal.Debug, "debug", false, "Enable debug mode")
+	cmd.SetVersionTemplate("docker-app version {{.Version}}\n")
```

Comment:
This is needed to make sure it always prints `docker-app`, not the binary name (e.g. `docker-app-darwin`). This could also be `Docker App` if that's the preferred name 🤷‍♂️